elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/models/Table.java

```diff
@@ -105,7 +105,12 @@ public Table(Class<?> cls, EntityDictionary dictionary) {
             dbConnectionName = ((FromSubquery) annotation).dbConnectionName();
         }
 
-        this.id = new TableId(this.name, this.version, dbConnectionName);
+        if (this.version != null && ! this.version.isEmpty()) {
+            this.id = this.name + "." + this.version;
```

Comment:
we dont need version in ID anymore

Reply:
yes. that require changes to Metadatstore o/w tests fail, will update after testing Metadatastore changes.